src/components/Checkout/Payment/Payment.js

```diff
@@ -1,44 +1,42 @@
 import React from 'react';
-import cx from 'classnames';
-
+import StripeCheckout from 'react-stripe-checkout';
+import axios from 'axios';
```

Comment:
You should use HttpService component

Reply:
?

Reply:
@Nazar-bnm I mean we have HttpService already implemented